data/saas/config/hubspot_config.yml

```diff
@@ -121,3 +120,41 @@ saas_config:
               configuration:
                 field: status
                 value: SUBSCRIBED
+    - name: users
+      requests:
+        read:
+          path: /settings/v3/users/
+          method: GET
+          query_params:
+            - name: limit
+              value: 100
+          param_values:
+            - name: placeholder
+              identity: email
+          postprocessors:
+            - strategy: unwrap
+              configuration:
+                data_path: results
+            - strategy: filter
+              configuration:
+                field: email
+                value:
+                  identity: email
+          pagination:
+            strategy: link
+            configuration:
+              source: body
+              path: paging.next.link
+        update:
+          path: /settings/v3/users/<userId>
+          method: PUT
+          body: |
+            {
+              <all_object_fields>
+            }
+          param_values:
+            - name: userId
+              references:
+                - dataset: hubspot_connector_example
+                  field: users.id
+                  direction: from
```

Comment:
Looking for some help here as I don't think there is anything to update when it comes to pii here. Should we remove this?